docs/resources/samples/get-table-data.md

```diff
@@ -0,0 +1,259 @@
+---
+title: 'Output Excel data as JSON'
+description: 'Learn how to output Excel table data as JSON to use in Power Automate.'
+ms.date: 03/18/2021
+localization_priority: Normal
+---
+
+# Output Excel table data (including hyperlink data) as JSON for usage in Power Automate
+
+Often it is beneficial to extract Excel table data as an array of objects (each item representing a row) in the form of JSON using Office Scripts. This helps with extracting the data from Excel in the same format that is visible to the user. Data can then be fed into other systems using Power Automate flows.
```

Comment:
```suggestion
Often, it is beneficial to extract Excel table data as an array of objects (each item representing a row) in the form of JSON using Office Scripts. This helps with extracting the data from Excel in the same format that is visible to the user. Data can then be fed into other systems using Power Automate flows.
```
Add comma after often when used as introductory word in a sentence. (Omit comma after often when used as a modifyer.)